Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM owncloud/base:bionic
+
+LABEL maintainer="ownCloud DevOps <devops@owncloud.com>" \
+  org.label-schema.name="ownCloud Server" \
+  org.label-schema.vendor="ownCloud GmbH" \
+  org.label-schema.schema-version="1.0"
+
+ADD owncloud-*.tar.bz2 /var/www/
+
+RUN find /var/www/owncloud \( \! -user www-data -o \! -group root \) -print0 | xargs -r -0 chown www-data:root && \
```

Comment:
don't we have an official docker that already does this ?

Reply:
Not needed - will drop this file ...